packages/components/pie-input/src/index.ts

```diff
@@ -14,6 +14,7 @@ import styles from './input.scss?inline';
 import {
     types, statusTypes, InputProps, InputDefaultPropertyValues,
 } from './defs';
+import 'element-internals-polyfill';
```

Comment:
Could you add a note to the readme similar to the modal one around the inclusion of this polyfill please?

Reply:
Hey Kevin! Happy to, although I cannot see any notes for modal in the readme or index file. Could you please point me to it?

Reply:
I thought i saw it there. If it's not there then dont worry for now. We can manually look through components later when removing the polyfills.